webhook_test.go

```diff
@@ -45,7 +45,7 @@ func TestValidateSignature_InvalidSignature(t *testing.T) {
 
 func TestValidateSignature_ValidSignature(t *testing.T) {
 	wh := webhook{Token: "abc123"}
-	err := wh.ValidateSignature([]byte("hello world"), "fcc98c5b4f306cfe6b5b8fcce03ddb33fc13ae6b")
+	err := wh.ValidateSignature([]byte("hello world"), "8c301acf7e955038b486de8f2a35f7f28bb5755fd1f77e1dbf9ef9e27713ad0d")
```

Comment:
Keen to know if these tests were working before @crlssn 

It seemed to me that the signature that it was matching was wrong and I'm assuming they were failing before?

Reply:
Yeah, they are failing in `master` as well.